lang/src/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransform.kt

```diff
@@ -433,6 +494,16 @@ internal class AstToLogicalVisitorTransform(
             )
         }
 
+    override fun transformExprCallWindow(node: PartiqlAst.Expr.CallWindow): PartiqlLogical.Expr =
+        PartiqlLogical.build {
+            id(
+                getUniqueWindowName(node,),
+                caseInsensitive(),
```

Comment:
This can likely be case sensitive.

Reply:
It can, but should we specify this to be case Sensitive breaks the semantics a little doesn't it?